tests/unit_tests/document_loader/parsers/test_pdf_parsers.py

```diff
@@ -53,12 +53,6 @@ def _assert_with_parser(parser: BaseBlobParser, splits_by_page: bool = True) ->
         assert metadata["page"] == 0
 
 
-@pytest.mark.requires("fitz")
```

Comment:
A copy of both deleted tests has been retained in integration test. I'll re-add these tests in a follow up PR when I do clean up.